src/Servers.jl

```diff
@@ -51,14 +54,9 @@ soon, it is closed and discarded, otherwise, the timestamp for the
 ip address is updated in the global cache.
 """
 function check_rate_limit(tcp, rate_limit::Rational{Int})
-    ip = Sockets.getsockname(tcp)[1]
-    rate = Float64(rate_limit.num)
```

Comment:
This also fixes the a bug if you specify `rate_limit = 3//2` or something (e.g. non-1 denominator).